apps/marketplace/pages/BriefPage.js

```diff
@@ -91,6 +93,17 @@ class BriefPage extends Component {
     window.scrollTo(0, 0)
   }
 
+  showTrainingResumesFileUpload = () => {
+    let showTraining = false
```

Comment:
you couldnt use a ternary here? (I feel like the guy from the Clean Code article)

Reply:
No fair call :) it can be made a lot leaner